spec/services/claims/fee_calculator/graduated_price_spec.rb

```diff
@@ -527,6 +527,32 @@
             it_returns 'a successful fee calculator response', amount: 8585.00
           end
         end
+
+        context 'scheme 12' do
+          let(:offence_band) { create(:offence_band, description: '1.1') }
+          let(:offence) { create(:offence, :with_fee_scheme_eleven, offence_band:) }
```

Comment:
Should this use the `:with_fee_scheme_twelve` trait, instead of `:with_fee_scheme_eleven`?